src/pages/lib-v1/storage/get-properties/q/platform/[platform].mdx

```diff
@@ -6,7 +6,3 @@ export const meta = {
 import js6 from '/src/fragments/lib-v1/storage/js/get-properties.mdx';
 
 <Fragments fragments={{ js: js6 }} />
-
-import flutter0 from '/src/fragments/lib/storage/flutter/get-properties.mdx';
-
-<Fragments fragments={{ flutter: flutter0 }} />
```

Comment:
Added by mistake at some point, Flutter does not expose get-properties in lib-v1 route.